Changelog.md

```diff
@@ -1,5 +1,28 @@
 ## Gazebo Math 7.x
 
+### Gazebo Math 7.5.0 (2024-06-18)
+
+1. Backport: Adding cone primitives.
+    * [Pull request #594](https://github.com/gazebosim/gz-math/pull/594)
+
+1. Enable 24.04 CI on harmonic
+    * [Pull request #590](https://github.com/gazebosim/gz-math/pull/590)
+
+1. Add package.xml
+    * [Pull request #581](https://github.com/gazebosim/gz-math/pull/581)
+
+1. bazel: correctly export license
+    * [Pull request #586](https://github.com/gazebosim/gz-math/pull/586)
+
+1. Add missing eigen3.hh header for bazel build
+    * [Pull request #585](https://github.com/gazebosim/gz-math/pull/585)
+
+1. Revert changes break ABI
+    * [Pull request #Revert changes break ABI](https://github.com/gazebosim/gz-math/pull/Revert changes break ABI)
+
+1. Expose non-const reference to edges
+    * [Pull request #Expose non-const reference to edges](https://github.com/gazebosim/gz-math/pull/Expose non-const reference to edges)
```

Comment:
Maybe we didn't merge this one correctly. This is the commit. Maybe include the commit here? 
https://github.com/gazebosim/gz-math/commit/1649f76ddee22971c10c9c4bd51d16653145f8b9